### DISCUSSION TITLE
Dukun Langit

### DISCUSSION THREAD

**Discussion by Deadnpretty** [thatskymod/Sky-CotL-Scripts#59] [Q&A]
Heyy i've been using canvas for quite a while now and i've noticed that players can't see my cosmetics anymore. I've also witnessed players appear with cosmetics then turn into moths next. Is there a way to fix this?

**Comment by thatskymod** [comment]
## Duplicate of #55.
### Closing as impossible to resolve.

The ability to fake or substitute spells or cosmetics was a feature in the past but is no longer supported due to restrictions imposed by thatgamecompany (i.e., the server now validates ownership of spells, preventing any possibility of spell or clothes forgery through client-side mods).

Thank you for your report, and have a great day!

**Reply by ImaGinedRa5ons** [reply]
the dunkun langit doesnt work anymore
